config/keycloak.php

```diff
@@ -3,7 +3,7 @@
 declare(strict_types=1);
 
 return [
-    'enabled' => env('KEYCLOAK_ENABLED', false),
+    'enabled' => env('KEYCLOAK_ENABLE', false),
```

Comment:
I guess the old name was better?

Reply:
Fine, but that does request an extra pr: https://github.com/cultuurnet/appconfig/pull/632